.gitlab/benchmarks/macrobenchmarks.yml

```diff
@@ -39,6 +39,7 @@ variables:
     DD_REMOTE_CONFIGURATION_ENABLED: "false"
```

Comment:
A bit unrelated but can we set `DD_RUNTIME_METRICS_ENABLED` to `false`. This feature is disabled by default.

Reply:
I can do it right now :+1: